qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html

```diff
@@ -1,5 +1,19 @@
 {% from qiita_pet.handlers.util import download_link_or_path %}
 
+<script>
+function parameters_changed(obj, ppd_id){
+  data = "foo"
```

Comment:
Do you really need to initialize this variable? If so, does it have to be "foo" or can it be simply ""?